roseus/CMakeLists.txt

```diff
@@ -126,12 +126,10 @@ execute_process(COMMAND cmake -E copy ${PROJECT_SOURCE_DIR}/bin/roseus ${CATKIN_
 install(PROGRAMS bin/roseus
   DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
 # install
-install(PROGRAMS bin/roseus
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 install(CODE "execute_process(COMMAND cmake -E make_directory \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION} RESULT_VARIABLE _mkdir_roseus_result OUTPUT_VARIABLE _mkdir_roseus_output)
               message(\"cmake -E make_directory \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION} returns \${_mkdir_roseus_result} ... \${_mkdir_roseus_output}.\")
-              execute_process(COMMAND cmake -E create_symlink ../${CATKIN_PACKAGE_BIN_DESTINATION}/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ RESULT_VARIABLE _install_roseus_result OUTPUT_VARIABLE _install_roseus_output)
-              message(\"create_symlink ../${CATKIN_PACKAGE_BIN_DESTINATION}/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ returns \${_install_roseus_result} ... \${_install_roseus_output}.\")")
+              execute_process(COMMAND cmake -E create_symlink ../${CATKIN_PACKAGE_SHARE_DESTINATION}/bin/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ RESULT_VARIABLE _install_roseus_result OUTPUT_VARIABLE _install_roseus_output)
+              message(\"create_symlink ../${CATKIN_PACKAGE_SHARE_DESTINATION}/bin/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ returns \${_install_roseus_result} ... \${_install_roseus_output}.\")")
```

Comment:
your original code
```
cmake -E create_symlink ../${CATKIN_PACKAGE_BIN_DESTINATION}/roseus roseus WORKING_DIRECTORY \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CATKIN_GLOBAL_BIN_DESTINATION}/ 
```
seems

1. you re on global bin location
2. try to create symlink from ../bin/roseus to roseus


what we have to do is

1. you're on shared location .. share/roseus/
2. creat esymlink from ../../bin/roseus to roseus,

fixed in 
https://github.com/k-okada/jsk_roseus/commit/4c250d613f1d28ebba79546a3837d2a1bcee9cfa